sunpy/time/timeformats.py

```diff
@@ -80,10 +80,10 @@ class TimeTaiSeconds(TimeFromEpoch):
     --------
     >>> from astropy.time import Time
     >>> t = Time('1958-01-01T00:00:00', format='isot', scale='tai')
-    >>> t.tai_seconds
+    >>> t.tai_seconds  # doctest: +SKIP
```

Comment:
We should open a tracking issue for when numpy 2.0 is released to un-skip these (and anything else which will need fixing).

Reply:
https://github.com/sunpy/sunpy/issues/7566